conf/routes

```diff
@@ -34,8 +34,10 @@ PUT            /api/contents/*path                       controllers.Application
 DELETE         /api/contents/*path                       controllers.Application.deleteNotebook(path:String)
 
 # The presentation parameter is optional. E.g. /notebooks/abc.snb?presentation=report
-GET            /notebooks/*snb                           controllers.Application.openNotebook(snb:String, presentation: Option[String])
+GET            /notebooks/$snb<.+\.snb>                  controllers.Application.openNotebook(snb:String, presentation: Option[String])
```

Comment:
> gotcha, however now the applied method sounds weird no? openNotebook

are you asking about this line? ~:ok: IMHO, this defined a regexp for `snb` file.